robocorp-code/src/robocorp_code/inspector/inspector_api.py

```diff
@@ -8,13 +7,14 @@
 from robocorp_ls_core.python_ls import PythonLanguageServer
 from robocorp_ls_core.robotframework_log import get_logger
 
+
 from robocorp_code.inspector.web._web_inspector import (
     PickedLocatorTypedDict,
     WebInspector,
 )
+from robocorp_code.inspector.windows.windows_inspector import WindowsInspector
```

Comment:
This change is the opposite of what I mentioned earlier.

i.e.: the idea is that the code should be at the top-level:

```
if typing.TYPE_CHECKING:
    from robocorp_code.inspector.windows.windows_inspector import WindowsInspector
    from robocorp_code.inspector.image._image_inspector import ImageInspector
```

and then only locally, where it's actually needed, local imports would be done (so, we don't even import it unless it's actually needed).

Reply:
I've tried that, but the execution fails. 
It seems that that IF isn't valid at runtime and `typing.TYPE_CHECKING` is false so the imports don't execute and the code breaks. 
That is why I reverted this and did direct imports.

Reply:
This is the missing:

> and then only locally, where it's actually needed, local imports would be done (so, we don't even import it unless it's actually needed).

part.

i.e.: you do that at top-level and then in the functions which actually use the `ImageInspector` you have to add the import again (but then, just inside that function).